### DISCUSSION TITLE
Integration with new 'Open Quickly' from FileMaker 21.1

### DISCUSSION THREAD

**Discussion by marcel-more** [fmIDE/fmIDE#40] [Ideas]
The new Open Quickly feature looks promising in the way that many objects inside the FileMaker development dialogs are now accessible by typing in it's name and hit enter.

There are now shortcuts to access 
- Layouts
- Scripts
- Tables
- Fields
- TOs

Two possible scenarios for fmeIDE:

1. simulate keystrokes in the GUI to navigate to named objects from fmIDE

2. drop a feature request to Claris to extend 'Open Quickly' for a programmatic approach without UI interaction (e.g. as an API call, CLI command or even a FileMaker script step)

This would allow straight access to objects inside a solution directly from fmIDE.

**Comment by mrwatson-de** [comment]
Hi Marcel,

thank you very much for your input, I'm so glad that you are actively using fmIDE!

Great ideas - Christian has also just added a shortcut key in MBS to get to the object tree search box - that is also a feature I want to integrate.

We can work on Claris to get some kind of programmatic access to the FileMaker IDE,… part of getting such a change through their Agile filter, will be promoting fmIDE more / showing what it can do.

We need to get more developers on board … and work on some SIMPLE impressive video / ads. I'm just working on getting my entire website rewritten and back online :)

P.S. Congratulations on the Stammtisch anniversary!

**Comment by stregatto-hub** [comment]
a shortcut to copy a xml snippet! :))

.g.

—————————————
Giulio Villani

800 98 53 30
***@***.***
www.fmguru.it
www.sviluppofmguru.it

FileMaker Certified Developer 7-19
FileMaker Certified Trainers
FileMaker Business Alliance/Claris Partner
Claris Connect Partner

Scopri il calendario dei nostri corsi FileMaker: https://www.fmguru.it/calendario-filemaker-fmguru/

**Reply by mrwatson-de** [reply]
Hi Giulio!

if you have fmAutoMate installed you can 'Xut', 'Xopy' and 'PaXte' FileMaker objects directly to XML! Just press **Ctrl**+Cmd+X, …C or …V!

Note: After restarting FileMaker you have to turn the fmAutoMate HotKeys back on … just right click on a script step and choose "ON - turn  fmAutoMate HotKeys on" from the fmAutoMate script step context menu.

see you at EngageU!

😉

**Reply by mrwatson-de** [reply]
And you can use those shortcuts with ANY viable FileMaker object! 😃
